pkgs/development/libraries/science/math/bonmin/default.nix

```diff
@@ -35,20 +39,34 @@ stdenv.mkDerivation rec {
     bzip2
     cbc
     clp
+    coin-utils
     ipopt
     lapack
     libamplsolver
+    osi
```

Comment:
Same as this change - it'd be nice to explain in a comment what do these new deps add.